website/source/docs/templates/engine.html.md

```diff
@@ -76,7 +76,7 @@ Here is a full list of the available functions for reference.
 
     Valid variables to request are: "ID", "Host",
     "Port", "User", "Password", "ConnType",
-    "PackerRunUUID", "PackerHTTPAddr", "SSHPublicKey", and "SSHPrivateKey".
+    "PackerRunUUID", "PackerHTTPAddr", "SSHPublicKey", "SSHPrivateKey", "SSHPublicKeyString", and "SSHPrivateKeyString".
```

Comment:
Is it documented anywhere that the value of SSHPrivateKey is in bytes? I wonder if we need to document the difference between these two variables.

Reply:
No, it's not  👀. I'm waiting for @SwampDragons to reply to my question in the issue because maybe I can make the original values as String instead of creating a new one. I don't know when the []byte is useful. Do you know?
If I can do this I can just leave it the way it is, right?

Reply:
Circling back here. I think it is in bytes solely for the input as it comes in from a file and passed around as a byte slice. But as for an output I agree having it as a string is better option. 

Re: I can just leave it the way it is. If you are thinking of changing the input I would say leave that as is and just cast the output to a string.

Reply:
Actually I wanted just to change the variable type as you said in the end.